apps/src/applab/designMode.js

```diff
@@ -398,17 +432,35 @@ designMode.configureDesignToggleRow = function () {
       studioApp, Applab.onDesignModeButton);
   var throttledDesignModeClick = _.debounce(designModeClick, 250, true);
 
-  // TODO (brent) - still need logic to generate list of screens, and rerender
-  // DesignToggleRow on changes
-  React.render(
-    React.createElement(DesignToggleRow, {
-      screens: ['screen1'],
-      onDesignModeButton: throttledDesignModeClick,
-      onCodeModeButton: Applab.onCodeModeButton,
-      handleManageAssets: showAssetManager
-    }),
-    designToggleRow
-  );
+  var firstScreen = $('.screen').first().attr('id');
+  designMode.changeScreen(firstScreen);
+};
+
+designMode.changeScreen = function (screenId) {
```

Comment:
what's the contract -- what happens if the screen doesn't exist? a comment would be helpful (apologies if this is explained somewhere else)